### DISCUSSION TITLE
executed queries in log after update

### DISCUSSION THREAD

**Discussion by 4tler** [nextcloud/all-in-one#1277] [Questions] [answered]
Hello, I get unfortunately after de change from beta to latest many messages in the log and my cloud runs very slowly.

The cloud is run in a LXC under Proxmox:
Ubuntu 22
Nextcloud AIO v2.0.4

active are:
Apache (Running)
Database (Running)
Nextcloud (Running)
Redis (Running)
Talk (Running)
Imaginary (Running)

I just recently switched from a natively installed NC to NC AIO. 

Here are the messages in the LOG where this message is just one of many that all show up in the same pattern.

```
[spreed] Warnung: Controller OCA\Talk\Controller\RoomController::getRooms executed 831 queries.

GET /ocs/v2.php/apps/spreed/api/v4/room?includeStatus=true
from 00.000.000.00 by username 2022-10-17T14:27:38+00:00
```

or this:

```
[notifications] Warnung: Controller OCA\Notifications\Controller\EndpointController::listNotifications executed 835 queries.

GET /ocs/v2.php/apps/notifications/api/v2/notifications
from 000.000.000.00 by username 2022-10-17T14:33:58+00:00
```
```
{"reqId":"FxPnKrXfKP9L6ExM5QMK","level":2,"time":"2022-10-17T14:35:00+00:00","remoteAddr":"00.000.000.00","user":"username","app":"spreed","method":"GET","url":"/ocs/v2.php/apps/spreed/api/v4/room?includeStatus=true","message":"Controller OCA\\Talk\\Controller\\RoomController::getRooms executed 1669 queries.","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:105.0) Gecko/20100101 Firefox/105.0","version":"24.0.6.1","data":{"app":"spreed"},"id":"634d681dbd57b"}
```

Iget these messages almost every minute from a lot of apps. Anyone have any ideas how to fix this?

**Comment by szaimen** [comment, marked as answer]
High, can you post your config.php file?

Most likely is the debug mode enabled which shouldn't on a productive instance and will remove these warnings.

**Reply by 4tler** [reply]
ok, that was indeed the problem for the messages. Thanks.
Now I only have the problem with the poor performance. The cloud is used by a maximum of 15 users at the same time and it feels like it gets slower with each additional user. i had this problem before. An update of the MariaDB was the solution. With the move to postgres, it seems to be back. At least it seems to be very similar to the problem at that time.

The logs only show a message for the [deck app](https://github.com/nextcloud/deck/issues/3358), which could not be solved yet.

Does anyone have any ideas where to start looking?


Hardware should not be a problem, the cloud has 8 cores and 8 GB RAM available.

**Reply by szaimen** [reply]
You can try to disable all optional things and see if it improves the performance.
